packages/ui/src/components/vuestic-components/va-toast/VaToast.vue

```diff
@@ -17,8 +17,7 @@
         <h2 v-if="title" class="va-toast__title" v-text="title"></h2>
```

Comment:
Component is done poorly:
- [ ] `window.setTimeout(` -> `setTimeout`
- [ ] `Reset timer if user press Delete or Backspace. I'm not sure we need this` - remove or figure it out.
- [ ] `onClose` handler is super buggy. Should be triggered only on manual click. Is being triggered on any close.
- [ ] Why do we have both `visible` and `closed` in component?
- [ ] Should support component or render (see [doc](https://docs.google.com/spreadsheets/d/1rOB9fOPQN1-06KKti542a1QtCeKnH81mR6Y1OyTGTAI/edit#gid=0)). Actually it seems to be supported from
![image](https://user-images.githubusercontent.com/5394573/103087234-d81dc380-45f7-11eb-8711-dc44dcfd47af.png), so maybe just docs + demo issue.
- [ ] `options.message = 'REPLACED_BY_VNODE'` looks weird. Can we remove that?

Reply:
I think it's ok if we could stay `onClose` handler on any close, not only after the manual close. Of course, we could provide a special variable to show if notification has been closed manually or not, but think it's not very helpful for end-users

Reply:
I would like to remove all `slots` things in here? Render method it looks more powerful and easy to use, can I?

Reply:
@damirJa yeah. Render is fine.